composer.json

```diff
@@ -16,33 +16,36 @@
         }
     ],
     "require": {
-        "php": "~5.5|~7.0",
-        "sonata-project/admin-bundle": "~3.0",
-        "sonata-project/block-bundle": "^2.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/property-access": "~2.2|~3.0",
-        "doctrine/phpcr-odm": "~1.1",
-        "doctrine/phpcr-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": "~2.0",
-        "symfony-cmf/resource-rest-bundle": "~1.0"
+        "php": "^5.5 || ^7.0",
+        "sonata-project/admin-bundle": "^3.0@dev",
+        "sonata-project/block-bundle": "^3.0@dev",
+        "symfony/framework-bundle": "^2.3 || ^3.0",
+        "symfony/property-access": "^2.2 || ^3.0",
+        "doctrine/phpcr-odm": "^1.1",
+        "doctrine/phpcr-bundle": "^1.1",
+        "symfony-cmf/tree-browser-bundle": "^2.0@dev",
+        "symfony-cmf/resource-rest-bundle": "^1.0@dev"
     },
     "require-dev": {
-        "jackalope/jackalope-jackrabbit": "~1.0",
-        "symfony/phpunit-bridge": "~2.7|~3.0",
-        "symfony/security-acl": "~2.4|~3.0",
-        "fabpot/php-cs-fixer": "~0.1|~1.0"
+        "symfony-cmf/resource-bundle": "@dev",
+        "symfony-cmf/resource": "@dev",
+        "dantleech/glob-finder": "@dev",
+        "puli/repository": "@beta",
+        "sonata-project/core-bundle": "@dev",
+        "jackalope/jackalope-jackrabbit": "^1.0",
+        "symfony/phpunit-bridge": "^2.7 || ^3.0",
+        "symfony/security-acl": "^2.4 || ^3.0",
+        "fabpot/php-cs-fixer": "^1.0"
     },
     "provide": {
         "sonata-project/admin-bundle-persistency-layer": "1.0.0"
     },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
     "autoload": {
         "psr-4": { "Sonata\\DoctrinePHPCRAdminBundle\\": "" }
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.0-dev"
+            "dev-master": "2.x-dev"
```

Comment:
I don't understand this change, but the change was part of #370 so I left it here.